seqr/views/utils/variant_utils.py

```diff
@@ -147,9 +147,12 @@ def get_phenotype_prioritization(families, gene_ids=None):
     data_by_individual_gene = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
 
     gene_filter = {'gene_id__in': gene_ids} if gene_ids is not None else {}
+    phenotype_genes = PhenotypePrioritization.objects.none()
+    for indiv in Individual.objects.filter(family__in=families):
+        phenotype_genes |= PhenotypePrioritization.objects.filter(individual=indiv, **gene_filter).order_by('rank')[:10]
```

Comment:
this contains many unnecessary queries. In general, you always want as few database queries as possible. In this case, instead of ordering by rank and picking the top 10 for each individual, why not just explicitly filter for rank:
```
PhenotypePrioritization.objects.filter(
  individual__family__in=families, rank__lte=10,  **gene_filter,
).order_by('disease_id')
```

Reply:
Oh, I didn't know that we could use the value of the `rank` field. But I'm curious since we never return records with a `rank` of more than 10, why do we still load them?